<launch>

    <node name="upsampling_kalman" pkg="linear_kalman" type="upsampling_kalman" output="screen">
        <param name="rate" value="100" />
        
        <param name="noise_meas_x" value="1.0" />
        <param name="noise_meas_y" value="1.0" />
        <param name="noise_meas_z" value="1.0" />

        <param name="noise_pos_x" value="10.0" />
        <param name="noise_pos_y" value="10.0" />
        <param name="noise_pos_z" value="10.0" />

        <param name="noise_vel_x" value="0.01" />
        <param name="noise_vel_y" value="0.01" />
        <param name="noise_vel_z" value="0.01" />
        
        <param name="pt1_filter" value="0.95" />

    </node>

</launch>
